spring-cloud-gcp-samples/spring-cloud-gcp-logging-sample/README.adoc

```diff
@@ -0,0 +1,37 @@
+= Spring Cloud GCP Logging Starter Example
+
+This sample application demonstrates using the
+link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-logging[Spring Cloud GCP Logging Starter] in your code.
+
+== Setup & Configuration
+1. Create a Google Cloud Platform Project
+1. Have the https://cloud.google.com/sdk/[Google Cloud SDK] installed and
+initialized and logged in with
+https://developers.google.com/identity/protocols/application-default-credentials[application
+default credentials].
+
+1. Check that the Stackdriver Logging API is enabled.
+
+== Run the Example
+Deploy the sample application from Maven:
```

Comment:
Maybe clarify that you're deploying to "App Engine flexible environment"

Reply:
done